### PR TITLE
Emb2016 dev

### PR DESCRIPTION
**Pull request by azotz**

### CODE REVIEW

File: python/ArtusConfigs/Run2Analysis/Includes/settingsJEC.py [hunk 1]
```diff
@@ -13,20 +13,20 @@ def __init__(self, nickname):
...
-		
+
 		self["JetEnergyCorrectionUncertaintySource"] = "Total"
 		#self["JetEnergyCorrectionUncertaintyShift"] = 0.0
-		
+
 		#All commented out
-		
+
 		if re.search("Run2015|Embedding2016", nickname):
```
Comment (jd550179) [inline python/ArtusConfigs/Run2Analysis/Includes/settingsJEC.py]: dooes embedding 2016 needs to be here?

File: python/ArtusConfigs/Run2CPStudies/tt.py [hunk 2]
```diff
@@ -162,9 +199,9 @@ def build_config(self, nickname, *args, **kwargs):                #Maybe change
 			self["NoHltFiltering"]= True
 			self["DiTauPairNoHLT"]= True
 		else:
-			self["NoHltFiltering"]=False
-			self["DiTauPairNoHLT"]= False	
-		
+			self["NoHltFiltering"]= True
```
Comment (jd550179) [inline python/ArtusConfigs/Run2CPStudies/tt.py]: should this be True?

File: python/plotting/configs/samples.py [hunk 3]
```diff
@@ -39,9 +39,10 @@ def get_config(self, samples, channel, category, nick_suffix="", postfit_scales=
 		weight = "(1.0)"
 		if len(weights) > 0:
 			weight = "(" + ("*".join(weights)) + ")"
-		
+		print samples
```
Comment (jd550179) [inline python/plotting/configs/samples.py]: remove print

File: python/plotting/configs/samples.py [hunk 4]
```diff
@@ -39,9 +39,10 @@ def get_config(self, samples, channel, category, nick_suffix="", postfit_scales=
...
 		for sample in samples:
 			config = sample(self, config, channel, category, weight, nick_suffix, **kwargs)
+			print config
```
Comment (jd550179) [inline python/plotting/configs/samples.py]: remove print

File: scripts/makePlots_controlPlots.py [hunk 5]
```diff
@@ -493,18 +493,19 @@ def blind_signal(config, blinding_threshold, ratio_true):
 						category = "_" + category if category else "",
 						quantity = quantity
 				)
+				print binnings_key
 				if binnings_key not in binnings_settings.binnings_dict and channel + "_" + quantity in binnings_settings.binnings_dict and "--x-bins" not in args.args:
 					binnings_key = channel + "_" + quantity
 				if binnings_key not in binnings_settings.binnings_dict:
 					binnings_key = None
-				
+				print binnings_key
```
Comment (jd550179) [inline scripts/makePlots_controlPlots.py]: print remove

File: python/ArtusConfigs/Run2Analysis/Includes/settingsJEC.py (hunk 1 above)
Comment (azotz) [reply python/ArtusConfigs/Run2Analysis/Includes/settingsJEC.py]: no, removed

File: python/ArtusConfigs/Run2CPStudies/tt.py (hunk 2 above)
Comment (azotz) [reply python/ArtusConfigs/Run2CPStudies/tt.py]: No

File: python/plotting/configs/samples.py (hunk 3 above)
Comment (azotz) [reply python/plotting/configs/samples.py]:
```suggestion
```

File: python/plotting/configs/samples.py (hunk 4 above)
Comment (azotz) [reply python/plotting/configs/samples.py]:
```suggestion
```

File: scripts/makePlots_controlPlots.py (hunk 5 above)
Comment (azotz) [reply scripts/makePlots_controlPlots.py]:
```suggestion
```

**Pull request merged**